src/index.html

```diff
@@ -163,66 +163,127 @@ <h1>Scaling <i>beyond</i> geospatial tech</h1>
         </div>
       </div>
     </section>
+    <section>
+      <div class="bg-background-blue">
+        <div class="max-width-container flex justify-center">
+          <div id="spec-button" class="sm:hidden text-white border-gray-button border-1 py-2 px-10 font-semibold">
```

Comment:
The button border in the designs is at 10% opacity.

```suggestion
          <div id="spec-button" class="sm:hidden text-white border-gray-button border-opacity-10 border-1 py-2 px-10 font-semibold">
```